### PR TITLE
person was not able to get Follow! maybe blocked

### PR DESCRIPTION
**Pull request by sionking**
Check by reload that person is not follow blocked.
I have seen follow block by follow button not changing to "following" but stay "Follow.
but for any case I also reload the page each 1-5 times..

### CODE REVIEW

**Comment by timgrossmann** [comment]
@sionking Looks good, this will probably make things more clear if we really encounter a follow block.

Thank you very much for this!

P.S. Let's wait for another approval before merging this 😉

**Comment by sionking** [comment]
lets wait for the crying babies get blocked :()

**Comment by sionking** [comment]
I need to return also "failed to retrieve following status" from check_following function, since it might be it that we failed to read the status and we think we got blocked.

**Comment by timgrossmann** [comment]
@sionking Haha, there might already be some in the issues 😅 

I think the second comment you made absolutely makes sense. We don't want the users to think they have been blocked, too often.

Thank you for your great additions

File: README.md [hunk 1]
```diff
@@ -71,6 +71,7 @@ Table of Contents
   * [Pick Nonfollowers of a user](#pick-nonfollowers-of-a-user)
   * [Pick Fans of a user](#pick-fans-of-a-user)
   * [Pick Mutual Following of a user](#pick-mutual-following-of-a-user)
+* [Third Party InstaPy GUI for Windows](#third-party-instapy-gui-for-windows)
```
Comment (timgrossmann) [inline README.md:74]: I think you quickly need to do a rebase on master before we merge this.

Comment (sionking) [reply README.md:74]: I am new to git baaa

Comment (timgrossmann) [reply README.md:74]: Should be solved by a rebase on InstaPy master:
https://gist.github.com/ravibhure/a7e0918ff4937c9ea1c456698dcd58aa

**Pull request closed without merging**